fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVUtil.java

```diff
@@ -17,6 +17,10 @@
 
 package org.apache.doris.mtmv;
 
+import org.apache.doris.analysis.DateLiteral;
```

Comment:
if u mixed-use same name class in analysis and Nereids, u should use simple name from one package to avoid typo. for this file, u should always use simple name for Nereids' expression, and use full-qualifier name for legacy planner's expression.